examples/textual_inversion/textual_inversion.py

```diff
@@ -191,6 +191,13 @@ def parse_args():
             "and an Nvidia Ampere GPU."
         ),
     )
+    parser.add_argument(
```

Comment:
This is because I can only get fp32 tensors in the output of unet, vae, text_encoder and loss even if I chose mixed_precision=bf16 in both accelerate config and command line. The output tensor of unet is bf16 when I apply autocast.   It might be my mistakes and I will check it. It would be nice if you can have a try to double check. Thanks.